ui/src/parts/lightbox.tsx

```diff
@@ -22,93 +22,83 @@ import { getColorFromString, isDark } from '@fluentui/react'
 export const lightboxB: Box<LightboxProps | null> = box()
 
 const
-  HEADER_HEIGHT = 40,
-  IMAGE_CAPTIONS_HEIGHT = 40, // Total height of title and decription.
-  FOOTER_VERTICAL_PADDING = 10,
-  IMAGE_NAV_HEIGHT = 148,
-  IMAGE_SIZE = 120,
-  ICON_SIZE = 38
+  IMAGE_CAPTIONS_HEIGHT = 46, // Total height of title and decription.
+  NAV_IMAGE_SIZE = 120,
+  IMAGE_NAV_HEIGHT = NAV_IMAGE_SIZE + 28, // 28 is the height of the scrollbar.
+  ICON_SIZE = 38,
+  ICON_MARGIN = 8,
+  IMAGE_NAV_BOTTOM_MARGIN = 8
 
 const
   css = stylesheet({
     body: {
       position: 'fixed',
       inset: '0px',
       zIndex: 1,
-      backgroundColor: 'rgba(0, 0, 0, 0.9)',
+      backgroundColor: 'rgba(0, 0, 0, 0.9)'
     },
     img: {
       position: 'absolute',
-      inset: '0px',
       margin: 'auto',
-      maxHeight: '100%',
-      maxWidth: '100%',
-    },
-    header: {
-      height: HEADER_HEIGHT,
-      textAlign: 'right'
-    },
-    content: {
-      position: 'relative',
+      left: 0,
+      right: 0,
+      top: ICON_SIZE + ICON_MARGIN * 2,
+      maxWidth: '100vw'
     },
-    footer: {
-      boxSizing: 'border-box',
-      padding: `${FOOTER_VERTICAL_PADDING}px 0`,
-      textAlign: 'center',
-      color: '#fff',
+    closeButton: {
+      position: 'absolute',
+      right: ICON_MARGIN,
+      top: ICON_MARGIN
     },
     imageNav: {
-      marginTop: 10,
+      position: 'absolute',
+      left: '50%',
+      bottom: IMAGE_NAV_BOTTOM_MARGIN,
       height: IMAGE_NAV_HEIGHT,
-      boxSizing: 'border-box',
-      overflow: 'auto',
-      whiteSpace: 'nowrap'
+      whiteSpace: 'nowrap',
+      overflowX: 'auto',
+      maxWidth: '100vw',
+      transform: 'translateX(-50%)'
     },
     navImg: {
+      height: NAV_IMAGE_SIZE,
+      width: NAV_IMAGE_SIZE,
       boxSizing: 'border-box',
       objectFit: 'cover',
-      height: IMAGE_SIZE,
-      width: IMAGE_SIZE,
       margin: '2px',
       cursor: 'pointer',
       filter: 'brightness(30%)',
       border: '2px solid #000'
```

Comment:
These 2 are overriden in the inline `imageHighlightStyle `. Do we need them in both places? Since they are static, I guess they should be removed from inline style and kept in this class declaration.

Reply:
They are overriden just in case when the image is hovered or currently selected:
```ts
className={clas(css.navImg, 'lazy', style({ $nest: { '&:hover': imageHighlightStyle } }))}
style={activeImageIdx === idx ? imageHighlightStyle : undefined}
```
where 
```ts
imageHighlightStyle = { filter: 'unset', border: '2px solid', borderColor: activeColor },
```
That means brightness is set back to `100%` by providing `unset` value and border changes color to `activeColor` which is computed dynamically based on the current theme.

Reply:
Makes sense.